docs/io/fits/index.rst

```diff
@@ -134,6 +134,37 @@ because by that point you are likely to run out of physical memory anyways), but
     ``del hdul[0].data``. (This works so long as there are no other references
     held to the data array.)
 
+.. _fits-cloud-files:
+
+Working with remote and cloud-hosted files
+""""""""""""""""""""""""""""""""""""""""""
+
+The :func:`open` function supports a ``use_fsspec`` argument which allows file
+paths to be opened using `fsspec <https://filesystem-spec.readthedocs.io>`__.
+The ``fsspec`` package supports a range of remote and distributed storage
+backends such as Amazon and Google Cloud Storage. For example, you can access a
+Hubble Space Telescope image located in the Hubble's public
+Amazon S3 bucket as follows:
+
+.. doctest-remote-data::
+    # Location of a large Hubble archive image in Amazon S3 (213 MB)
+    uri = "s3://stpubdata/hst/public/j8pu/j8pu0y010/j8pu0y010_drc.fits"
+
+    # Extract a 10-by-20 pixel cutout image
+    with fits.open(uri, use_fsspec=True, fsspec_kwargs={"anon": True}) as hdul:
+        cutout = hdul[1].section[10:20, 30:50]
```

Comment:
Ohhh, thank you.  It rendered correctly so I didn't think the `>>>`'s were necessary, but I will add them!

Reply:
yeap, rendering is a different beast, sphinx is happy to render after the `::` or directive, but pytest is more picky for proper indications for python snippets. None of these would fail, and none of the would be picked up as a doctest.